package.json

```diff
@@ -79,7 +79,6 @@
         "@vvo/tzdb": "^6.125.0",
         "args-parser": "~1.3.0",
         "axios": "~0.28.1",
-        "axios-ntlm": "1.3.0",
```

Comment:
Not going to maintain, but it was hard coded to `1.3.0` already. Technically no different.

I am doing this because of vulnerabilities in its axios (0.21.x).

Dropping it should prevent more users to upgrade to 2.0.0, so I do not want to. It is also not my plan.